frameit/lib/frameit/runner.rb

```diff
@@ -41,8 +41,13 @@ def run(path, color = nil)
             screenshot = Screenshot.new(full_path, color)
             screenshot.frame!
           rescue => ex
-            UI.error(ex.to_s)
-            UI.error("Backtrace:\n\t#{ex.backtrace.join("\n\t")}") if FastlaneCore::Globals.verbose?
+            # We don't want to show an error message for
+            #   [23:47:09]: Unsupported screen size [1295, 2590] for path './IMG_2343_framed_black.png'
```

Comment:
Isn't this an error message from `deliver`?
https://github.com/fastlane/fastlane/blob/c79cb094721dd2e0f49f4eb11aca36adf3f73666/deliver/lib/deliver/app_screenshot.rb#L304

We are already skipping files ending with `_framed.png` in line 29 of this file, so they don't get framed _again_. 

Is there an issue for this actual problem? My guess is that someone tried to upload a framed, but un-backgrounded screenshot (`frameit` was used to add a frame, but not a background/title/keyword) which doesn't resize the screenshot to App Store compatible sizes.